tests/ci_tests/code_formatter/crewai/test_crewai_main.py

```diff
@@ -0,0 +1,453 @@
+"""Tests for coffee_maker/code_formatter/crewai/main.py"""
+
+from unittest import mock
+
+import pytest
+from crewai import Agent
+
+from coffee_maker.code_formatter.crewai.main import _get_pr_file_content, run_code_formatter
+
+
+@pytest.fixture
+def mock_agents():
+    """Create real Agent instances for testing"""
+    return {
+        "senior_engineer": Agent(role="Senior Engineer", goal="Refactor code", backstory="Expert", verbose=False),
+        "pull_request_reviewer": Agent(role="Reviewer", goal="Review PR", backstory="Expert", verbose=False),
```

Comment:
Add a docstring for the test class
```suggestion
    "Tests for _get_pr_file_content helper function"

    def test_missing_github_token(self, monkeypatch, caplog):
        "Test that missing GITHUB_TOKEN is handled"
        monkeypatch.delenv("GITHUB_TOKEN", raising=False)

        result = _get_pr_file_content("owner/repo", 123, "src/file.py")

        assert result is None
        assert "GITHUB_TOKEN environment variable is not set" in caplog.text
```